package.json

```diff
@@ -38,24 +38,24 @@
   "dependencies": {
     "co": "^4.6.0",
     "js-combinatorics": "^0.5.2",
-    "lodash": "^4.16.6",
-    "mz": "^2.5.0",
-    "request-promise": "^4.1.1",
+    "lodash": "^4.17.4",
+    "mz": "^2.6.0",
+    "request-promise": "^4.2.0",
     "require-dir": "^0.3.1",
-    "uuid": "^2.0.3",
-    "winston": "^2.3.0"
+    "uuid": "^3.0.1",
+    "winston": "^2.3.1"
   },
   "devDependencies": {
     "chai": "^3.5.0",
-    "chai-as-promised": "^5.3.0",
-    "co-mocha": "^1.1.3",
-    "coveralls": "^2.11.14",
-    "eslint": "^3.5.0",
-    "eslint-plugin-standard": "^2.0.0",
-    "mocha": "^3.0.2",
-    "nyc": "^8.3.0",
-    "sinon": "^1.17.6",
-    "sinon-chai": "^2.8.0"
+    "chai-as-promised": "^6.0.0",
+    "co-mocha": "^1.2.0",
+    "coveralls": "^2.13.0",
+    "eslint": "^3.19.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "mocha": "^3.2.0",
+    "nyc": "^10.2.0",
+    "sinon": "^2.1.0",
+    "sinon-chai": "^2.9.0"
   },
   "engines": {
     "node": ">=5.0.0"
```

Comment:
Does it then mean that we are restricting the usage of our library to only those users who have latest NodeJS (https://docs.npmjs.com/files/package.json#engines)?